### DISCUSSION TITLE
When to support the new commands added in Redis 6.2?

### DISCUSSION THREAD

**Discussion by nigelis** [redis/go-redis#1738] [General]
Hi Maintainers,

I found that some new commands added in Redis 6.2 stable are not supported in [go-redis/redis](https://github.com/go-redis/redis). For example, [SMISMEMBER](https://redis.io/commands/smismember) is available since 6.2.0, and is kind of attractive in some scenarios, but it is not supported now.

So is there any plan or guideline to support these new commands?
May I contribute just like other projects, without special guidelines?

**Comment by nigelis** [comment]
Here is the corresponding release notes of redis: https://raw.githubusercontent.com/redis/redis/6.2/00-RELEASENOTES

**Reply by vmihailenco** [reply]
There are no special rules or guidelines on adding new commands. Just send a PR (with a test) or file an issue asking someone to add support for the command.
